packages/server/src/routes/index.ts

```diff
@@ -48,6 +49,7 @@ router.use('/components-credentials', componentsCredentialsRouter)
 router.use('/components-credentials-icon', componentsCredentialsIconRouter)
 router.use('/chatflows-uploads', chatflowsUploadsRouter)
 router.use('/credentials', credentialsRouter)
+router.use('/documentStore', documentStoreRouter)
```

Comment:
should we follow the naming convention `document-store`

Reply:
done